src/coreclr/nativeaot/System.Private.CoreLib/src/System/Threading/SyncTable.cs

```diff
@@ -51,7 +52,7 @@ internal static class SyncTable
         /// </summary>
 #if DEBUG
         // Exercise table expansion more frequently in debug builds
-        private const int InitialSize = 1;
+        private const int InitialSize = 2;
```

Comment:
The valid indices start at 2 is so that we could return 0 and 1 as true/false and the rest to mean a syncblock index for the slow path to retry with.